### PR TITLE
Reduce alerts on financial form

### PR DESCRIPTION
**Pull request by andrewdds**
This cleans up the alerts display on the financial verification form.

- The blue info alert only shows when there are no errors, or the extended form is not showing.
- The red error alert only shows if the missing TO isn't the only error
- The extended TO alert content is modified to reflect the instructions more clearly, rather than just duplicating the missing TO message. Also, helpdocs link is added to this alert.

screen shot 2018-10-02 at 8 40 10 am

screen shot 2018-10-02 at 8 40 17 am

### CODE REVIEW

File: templates/requests/financial_verification.html [hunk 1]
```diff
@@ -21,8 +21,8 @@
 <div class="col">
 
   {% if extended %}
-    {{ Alert('Task Order not found in EDA',
-      message="We could not find your Task Order in the EDA, our system of record. Please confirm that you have entered the information correctly and resubmit or click on the button below to enter the TO manually.",
+    {{ Alert('Manually enter Task Order information',
+      message="Additional field are displayed below, where you can manually enter financial information as documented in your Task Order.",
       level='warning'
```
Comment (dandds) [inline templates/requests/financial_verification.html]: This text is much better. I wonder if this should still be a warning, though. What do you think?

Comment (andrewdds) [reply templates/requests/financial_verification.html]: Mainly want to keep they yellow to visually tie it to the extra financial fields

File: atst/forms/financial.py [hunk 2]
```diff
@@ -151,6 +151,10 @@ def perform_extra_validation(self, existing_request):
     def is_missing_task_order_number(self):
         return "task_order_number" in self.errors
 
+    @property
+    def is_only_missing_task_order_number(self):
+        return "task_order_number" in self.errors and len(self.errors) == 1
```
Comment (andrewdds) [inline atst/forms/financial.py]: Is there a better way to do this?

Comment (patricksmithdds) [reply atst/forms/financial.py]: I, too, think its fine. Only other way I can think of is `self.errors.keys() == ['task_order_number']`, but I think your way is easier to read.

File: templates/requests/financial_verification.html [hunk 3]
```diff
@@ -21,8 +21,8 @@
 <div class="col">
 
   {% if extended %}
-    {{ Alert('Task Order not found in EDA',
-      message="We could not find your Task Order in the EDA, our system of record. Please confirm that you have entered the information correctly and resubmit or click on the button below to enter the TO manually.",
+    {{ Alert('Manually enter Task Order information',
+      message="Additional field are displayed below, where you can manually enter financial information as documented in your Task Order.",
```
Comment (patricksmithdds) [inline templates/requests/financial_verification.html]: "field" should be pluralized in the start of the message: "Additional fields are displayed below"

Comment (andrewdds) [reply templates/requests/financial_verification.html]: 👍

**Pull request merged**